clippy_lints/src/formatting.rs

```diff
@@ -46,12 +46,33 @@ declare_lint! {
     "suspicious formatting of `else if`"
 }
 
+/// **What it does:** Checks for possible missing comma in an array. It lints if
+/// an array element is a binary op and it lies on two lines.
```

Comment:
The only operators that could actually cause problems are `+-*` (Rust does not have unary `+` but people might think it does).